crates/router/src/scheduler/consumer.rs

```diff
@@ -82,14 +84,17 @@ pub async fn start_consumer(
                     workflow_selector,
                 ));
             }
-            Ok(()) | Err(oneshot::error::TryRecvError::Closed) => {
+            Ok(()) | Err(mpsc::error::TryRecvError::Disconnected) => {
                 logger::debug!("Awaiting shutdown!");
+                ////can panic!
+                rx.close();
```

Comment:
Not sure, if this panics. Can you review the docs and remove this comment, if not necessary

Reply:
Not required as it is closed already